runtime/vm/ContinuationHelpers.cpp

```diff
@@ -255,6 +255,8 @@ yieldContinuation(J9VMThread *currentThread, BOOLEAN isFinished)
 	if (!isFinished) {
```

Comment:
now that we have 'else' clause, we could invert the logic, but really up to you

Reply:
updated